imports/plugins/core/orders/server/no-meteor/queries/orders.js

```diff
@@ -9,15 +9,36 @@ import ReactionError from "@reactioncommerce/reaction-error";
  * @param {Object} context - an object containing the per-request state
  * @param {Object} params - request parameters
  * @param {String} params.accountId - Account ID to search orders for
- * @param {String} params.orderStatus - Workflow status to limit search results
- * @param {String} params.shopIds - Shop IDs for the shops that owns the orders
+ * @param {Object}  params.filters - Filters to apply to a list of orders
+ * @param {Array.<String>} params.shopIds - Shop IDs for the shops that owns the orders
  * @returns {Promise<Object>|undefined} - An Array of Order documents, if found
  */
-export default async function orders(context, { filter, orderStatus, shopIds } = {}) {
+export default async function orders(context, { filters, shopIds } = {}) {
   const { collections, shopsUserHasPermissionFor, userHasPermission } = context;
   const { Orders } = collections;
 
-  let query = {};
+  const query = {};
+  let dateRangeFilter = {};
+  let fulfillmentStatusFilter = {};
+  let paymentStatusFilter = {};
+  let searchFieldFilter = {};
+  let statusFilter = {};
+
+
+  // Add a date range filter if provided, the filter will be
+  // applied to the createdAt database field.
+  if (filters.dateRange && filters.dateRange.createdAt) {
+    const { createdAt } = filters.dateRange;
+    // Both fields are optional
+    const gteProp = createdAt.gte ? { $gte: createdAt.gte } : {};
+    const ltProp = createdAt.lt ? { $lt: createdAt.lt } : {};
```

Comment:
I think this should be `lte` as because `lt` of `2019-09-16T23:59:59.999Z` is `2019-09-16T23:59:59.998Z`. Plus, it matches the inclusiveness of `gte`.

Reply:
make sense.

Reply:
everywhere it's `lt` , `ltProp` here and in the schema should be updated to match `lte`, etc as well.